spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Inception_v1.scala

```diff
@@ -173,7 +174,7 @@ object Inception_v1_NoAuxClassifier {
       .setName("loss3/classifier").inputs(view)
     val loss = LogSoftMax().setName("loss3/loss3").inputs(classifier)
 
-    val model = Graph(input, loss)
+    val model = if (isDynamic) Graph.dynamic(input, loss) else Graph(input, loss)
```

Comment:
Why do we need isDynamic for the models/examples? It should always be static graph, no?

Reply:
This is for test... We have a set of test for these graph models, so I want to test both dynamic and static